30-indoor-air-quality-and-gas-leak/README.md

```diff
@@ -17,65 +18,26 @@ Monitor air quality, including Air Quality Index, CO2 and VOC concentration, and
 
 Sign up for a free account on [notehub.io](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-a/#set-up-notehub).
 
-## Sparrow Setup
-
-### Quickstart
-
-Follow the [Sparrow Quickstart](https://dev.blues.io/quickstart/sparrow-quickstart/) pair your Sparrow reference node with the gateway and associate it with the Notehub project you just created. Note also that you only need one reference node for this project, so you don't need to pair both nodes that came with the dev kit. Make sure that you use the ProductUID generated in [Notehub Setup](#notehub-setup) when it comes time to issue the [`hub.set`](https://dev.blues.io/api-reference/notecard-api/hub-requests/#hub-set) command in the quickstart.
-
-After you've completed the quickstart, leave the Notecarrier and Sparrow Essentials board powered and connected. These two devices will act as our gateway to Notehub, and you won't need to touch them again. The rest of this guide will focus on the Sparrow Reference Sensor node.
+## Swan Setup
 
 ### Air Quality Sensor Connection
 
-This solution makes use of the ENS160 sensor board, which measures indoor air quality. Connect the ENS160 sensor to the Sparrow Reference Sensor board using the Qwiic cable:
+This solution makes use of the ENS160 sensor board, which measures indoor air quality. Connect the ENS160 sensor to the Swan board using the Qwiic cable:
 
-1. Insert one end of the Qwiic cable into the port labelled Qwiic on the Sparrow Reference Sensor board
+1. Insert one end of the Qwiic cable into the port labelled Qwiic on the Swan board
 2. Insert the other end of the Qwiic cable into either of the Qwiic sockets on the ENS160 sensor board.
```

Comment:
It’d be nice to have one picture in here that shows everything hooked up for people to reference.